neurons/validator.py

```diff
@@ -635,6 +676,124 @@ def get_positions(self, synapse: template.protocol.GetPositions,
         synapse.error_message = error_message
         return synapse
 
+    # Enable validator <-> validator p2p checkpoint sync
+    def checkpoint_thread(self):
+        """
+        set up event loop for asynchronous sending of checkpoints
+        """
+        loop = asyncio.new_event_loop()
+        asyncio.set_event_loop(loop)
+        try:
+            bt.logging.info("calling send_checkpoint")
+            loop.run_until_complete(self.send_checkpoint())
+        finally:
+            loop.close()
+
+    async def send_checkpoint(self):
```

Comment:
Make a new class p2pSyncer to ensure validator.py stays as clean as possible. Similar to the PositionSyncer class